### DISCUSSION TITLE
Most compatible Doorbells (Poe/Ethernet/ONVIF)

### DISCUSSION THREAD

**Discussion by strarsis** [blakeblackshear/frigate#3572] [Ask A Question]
What Doorbell has PoE/Ethernet/ONVIF and is known to be exceptionally well supported by Frigate?
And as I am not picky enough already, it would be great if it costs less than 200 bucks.

I want to achieve all the nice stuff with that doorbell, independent from plans/3rd party cloud, with AI person detection, custom actions (e.g. Home Assistant) that triggers a notification on the smartphone and/or a ringer or shows the stream on (cheap) tablets mounted to the walls. I would also get hardware acceleration for the inference, like a Coral or a TPU or embedded GPU/accelerator SoC.

**Comment by NickM-27** [comment]
Readding for clarity from comment in other thread. I use the AD410 and it is easily supported with streaming within frigate, HA also has multiple ways to integrate which support doorbell press, changing options, etc. locally without any Amcrest app or account

https://github.com/rroller/dahua

For playing audio with the camera https://github.com/rroller/dahua/issues/181#issuecomment-1148628524 can be used until support is added to the integration

**Comment by strarsis** [comment]
@NickM-27: That's a nice doorbell (`AD410`). But I also want PoE/Ethernet (no WiFi) and it doesn't seem that it supports Ethernet at all.
Poe/Ethernet would be important for me as there is no existing wiring for powering it, the doorbell would be some distance away from any networking and I generally want to avoid WiFi.

**Reply by NickM-27** [reply]
May want to look into the upcoming reolink doorbell then, although reolonk support for frigate streams is more temperamental / finicky (depends on the camera)

**Reply by kdill00** [reply]
most doorcams that have POE are going to have some contruction/wall cutting to size it in as most are designed to just attach to that small 12v connection behind the current doorbell.  Under 200 you might be able to ebay something used.  Doorbird i believe has POE solutions but they are usually 300 to 600.  Most of the cheaper doorbell cams want you to pay for monthly garbage to get certain features.  If you have a 3d printer or use one of those sites that will print your projects for you, you could build a nice one as I did for my last place using a Raspi Zero 2 and a POE hat.  I even used an ESP32 cam which works great as long as their is some light.  I upped it to a zero 2 to use a better lense with automatic IR switching.  All that can be done for 20 bucks.  I also have an ethernet cable run to my doorbell but I have not used it at this house, I went the route of putting a camera up high behind you as you enter the porch, frigate and homeassistant to push a notification, as well switch my hdmi matrix for all devices to view the camera on all TVs in the house(delayed 10seconds to capture event) for a 15 second glimpse before turning them off or back to previous media.  If the intention is to have some sort of face detection opening locks and what not, a doorbell cam will be useful but my problem was they are not positioned well to see the entire area such as the lazy amazon deliverer who puts the package 20 steps from the door or a fast moving target across.  Sorry, thats a lot of words.   Let me know if you have anymore questions, I looked hard at this for awhile last year and most POE solutions are 500 dollars+.

**Reply by strarsis** [reply]
@kdill00: You are very knowledgeable and probably already have a quite sophisticated setup. How are you doing the Ethernet wiring in and around the house? I plan to have Ethernet/Fiber in conduits and on wall boxes, so the Frigate/NVR/NAS/Router equipment can all be cleanly and easily connected. I want to isolate the outdoor Ethernet using VLAN and a separate network that ends as Frigate/NVR box somehow, so everything else is viewed as client from the outside, even on the LAN.
Ideally all the processing/house intelligence stuff ends up in the basement or a closet where it runs quietly (ideally passively cooled/solid state) until something has to be upgraded.

The idea of building a doorbell oneself is a great one. Though I need something now that works reasonably well in the future. I had a very bad experience with WLAN doorbells with batteries, I had to use a WLAN antenna(!) AP, had to change the batteries very often (unscrewing the retaining screw, sliding it out of its holder, opening it up, taking out the batteries, putting new in, closing everything, hopefully it is still connecting afterwards, sliding it back into its holder, then finally screwing the retaining screw back). Also I don't like so much outside WLAN and I like the rock-solid power and data connection from PoE-Ethernet. There are many more expensive doorbells out there, but I am never sure whether they are really "open" and ONVIF and can be used by any software.
A doorbell "that works" for opensource home automation for 200-300 bucks would also be OK for me.

**Reply by kdill00** [reply]
A lot of the POE are proprietary for the classic "dealer home automation"  because they pretty much need to be installed as the house is being built.  I am lucky that I was able to build this house and had a CAT6 and CAT5 run to everyroom and area possible all heading back into a server closet.  I rely heavily on WiFI for most of my home automation besides the TVs which are all wired in as well as the HDMI matrix extenders ect.  As long as you have access points around your house that are wired in and not just "mesh" or relaying the signal around.  If the house is existing, without wire laid currently, you probably need to survey your attic and see if its even possible to get where you need to go.  Their are cheap fiber glass tent pole like poles that are long and you can drill from the attic down then use the pole to jimmy on down the wall space.  Probably need 2 people unless its easy access from top and bottom.  But I have 8 access points in various places around my house.  The cameras are all wired in, I tossed my NVR I originally bought that the 12 cams ran into and had a someone private vlan.  I havent locked it down again yet since hooking them all to a POE router.  You are right to try and avoid wifi for cameras if you can, but I will say I have 4 amcrest 80 dollar ptz wifi cameras, two for my garage and two at a business, and have been pleasantly surprised with them and their function.  Its been 9 months since I last had connection issues with it.  I guess the biggest question is do you have wire laid already?  Is this an existing house?  One thing I played with thats come a long way and works pretty well now is ethernet over your power outlets.  Gbps speeds are easily achievable now on those types of connections and you could use it to bring a wifi access point closer to where you need to be.  Is there not a regular door bell at the door with 12v-24v connection behind it?  How was the plane jane doorbell setup?  Yeah, I NEVER do battery for cameras as it just seems like an important event could someday be missed and the recharging ect.  I use batteries only for RF home automation devices, such as the water meter I use.

**Reply by kdill00** [reply]
As for the NVR depending on the number of cameras, i wouldnt do an actual nvr.  They are insanely expensive for practically no reason.  You could just tag your packets in a private vlan to accomplish basically the same thing.  I bought a Dell poweredge R620 with 4 network ports and at my business I route the cameras there through one of those ports set to local only which then I view on a remote homeassistant connected with my houses homeassistant. At home I have a 630 but my camera system is not as walled off as id want.    The poweredge has 10 HD slots and i crammed 8 5tb into it and 2 nvme cards and I run 22 cameras and a plex server from that.  As the poweredge r620 and 630 have enought space for an nvidia quadro p2000, I grabbed one of those for decoding/encoding for the cams and plex.  You can pick these servers up for 2 to 4 hundred.  I have an extra r620 with 120gb that I upgraded a few months ago and im looking to rid myself of it.  Only problem is you need space for a 1U server as I have a rack, I went that route.

**Reply by strarsis** [reply]
@kdill00: The server rack thing is a very nice idea, I will look into that. Also the PowerLine solution is a good alternative to making extra conduits for cables in the rooms, can use that money then for the actual equipment.

Are you using a TPU/Coral card for speeding up inference for Frigate?

**Reply by Gfermoto** [reply]
Dahua 2111

**Comment by twctg81** [comment]
Dahua VTO doorbell. Supports POE and has home assistant integration.

**Reply by strarsis** [reply]
@twctg81: There are multiple models for that line from what I see. Any particular model that is great?

**Reply by r3wt** [reply]
@twctg81 looks like this one maybe? https://www.dahuasecurity.com/ceen/products/All-Products/Video-Intercoms/IP-Series/Villa-Door-Station/Lite-Series/VTO2111D-P-S2

**Reply by twctg81** [reply]
https://www.dahuasecurity.com/products/All-Products/Video-Intercoms/IP-Series/Villa-Door-Station/Pro-Series/VTO2202F-P
